controllers/activegate/capability/capability.go

```diff
@@ -67,6 +94,34 @@ func NewKubeMonCapability(crProperties *dynatracev1alpha1.CapabilityProperties)
 			Configuration: Configuration{
 				ServiceAccountOwner: "kubernetes-monitoring",
 			},
+			initContainersTemplates: []v1.Container{
+				{
+					Name:            "certificate-loader",
+					ImagePullPolicy: v1.PullAlways,
+					WorkingDir:      "/var/lib/dynatrace/gateway",
```

Comment:
Wouldn't it make sense to have this as a const since we reuse it?

Reply:
Strings moved to consts